src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs

```diff
@@ -163,6 +166,121 @@ private sealed class EnhancedColorApplier
+
+            // Dark Theme
+            // We also check OperatorOverloaded and ControlKeyword for whether they are the PlainText color.
+            // This is because when the "Use Defaults" is invoked from the Fonts and Colors options page the
+            // color reported back will be the PlainText color since these Classifications do not currently have
+            // colors defined in the PKGDEF. The other identifier types do not either but the Identifier color
+            // happens to be the same as PlainText so an extra check isn't necessary. StructName doesn't need
+            // an additional check because it has a color defined in the PKGDEF. The Editor is smart enough
+            // to follow the BaseClassification hierarchy and render the colors appropriately.
+            private static readonly ImmutableDictionary<uint, ImmutableArray<string>> DarkThemeDefaults =
+                new Dictionary<uint, ImmutableArray<string>>()
+                {
+                    [DarkThemePlainText] = new List<string>()
+                    {
+                        PlainTextClassificationTypeName,
+                        ClassificationTypeNames.ConstantName,
+                        ClassificationTypeNames.ControlKeyword,
+                        ClassificationTypeNames.EnumMemberName,
+                        ClassificationTypeNames.EventName,
+                        ClassificationTypeNames.ExtensionMethodName,
+                        ClassificationTypeNames.FieldName,
+                        ClassificationTypeNames.Identifier,
+                        ClassificationTypeNames.LabelName,
+                        ClassificationTypeNames.LocalName,
+                        ClassificationTypeNames.MethodName,
+                        ClassificationTypeNames.NamespaceName,
+                        ClassificationTypeNames.OperatorOverloaded,
+                        ClassificationTypeNames.ParameterName,
+                        ClassificationTypeNames.PropertyName,
+                    }.ToImmutableArray(),
+                    [DarkThemeClass] = new List<string>()
+                    {
+                        ClassificationTypeNames.ClassName,
+                        ClassificationTypeNames.DelegateName,
+                        ClassificationTypeNames.ModuleName,
+                        ClassificationTypeNames.StructName,
+                    }.ToImmutableArray(),
+                    [DarkThemeEnum] = new List<string>()
+                    {
+                        ClassificationTypeNames.EnumName,
+                        ClassificationTypeNames.InterfaceName,
+                        ClassificationTypeNames.TypeParameterName,
+                    }.ToImmutableArray(),
+                    [DarkThemeOperator] = new List<string>()
+                    {
+                        ClassificationTypeNames.Operator,
+                        ClassificationTypeNames.OperatorOverloaded,
```

Comment:
Yes. In the comment above this map, there are cases where the color reported will be the plain text color since we aren't exporting a default color for OperatorOverloaded. However, after visiting Fonts and Colors it will be updated based on the classification hierarchy and report as the Operator color.